File: single-language-analysis/dataset-files/java/TestSmell072.java
// Original URL: https://github.com/yuchting/yuchbox/blob/c4e3d7c87a47f192a4a7632fbb91d68fb3dbd719/server/src/weibo4j/WeiboTest.java#L231-L237

public class TestSmell072 {
	@Test
	public void testFriendship() throws WeiboException, InterruptedException {
		assertNotNull(weibo.createFriendship("1646678371"));
		Thread.sleep(1000);
		assertNotNull(weibo.destroyFriendship("1646678371"));
		assertNotNull(weibo.existsFriendship("1377583044", "1646678371"));
	}
}

File: single-language-analysis/dataset-files/python/test_smell_072.py
import pytest

class TestWeibo:
    def test_smell_072(self):
        assert weibo.create_friendship("1646678371") is not None
        time.sleep(1)
        assert weibo.destroy_friendship("1646678371") is not None
        assert weibo.exists_friendship("1377583044", "1646678371") is not None